src/behaviors/constants.js

```diff
@@ -0,0 +1,7 @@
+export const EDITOR_COMPONENT = 'simpla-img-editor.html';
```

Comment:
I realise it's kind of random having these in here and not others - got halfway through moving all constants to this file and realised that might not be the best idea anyway. Wasn't sure where to put `DEFAULT_SRC` other than here, and seemed a bit daft having a constants file with one constant.

What say you @bedeoverend - move all constants here, or move `DEFAULT_SRC` elsewhere?